### PR TITLE
Fix lint errors so we can eventually run eslint on travis.

### PR DESCRIPTION
**Pull request by picklesrus**
Thanks for submitting code to Blockly!  Please fill out the following as part of your pull request so we can review your code more easily.

## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Reason for Changes

Fixes lint errors! (but not all of them)

### Test Coverage

Tested on:
- [ ] Desktop:
  - [x] Chrome
  - [ ] Firefox
  - [ ] Safari
  - [ ] Opera
  - [ ] IE 10+
  - [ ] IE 11
  - [ ] EDGE

- [ ] Smartphone/Tablet/Chromebook (please complete the following information):

### CODE REVIEW

File: core/toolbox.js [hunk 1]
```diff
@@ -296,8 +296,9 @@ Blockly.Toolbox.prototype.populate_ = function(newTree) {
 /**
  * Sync trees of the toolbox.
  * @param {!Node} treeIn DOM tree of blocks.
- * @param {!Blockly.Toolbox.TreeControl} treeOut
- * @param {string} pathToMedia
+ * @param {!Blockly.Toolbox.TreeControl} treeOut The TreeContorol object built
+ *  from treeIn.
```
Comment (rachel-fenichel) [inline core/toolbox.js]: Indent an additional 3 spaces (not managed by eslint)

File: core/field.js [hunk 2]
```diff
@@ -527,7 +527,8 @@ Blockly.Field.prototype.setValue = function(newValue) {
  * @param {!Event} e Mouse down event.
  * @private
  */
-Blockly.Field.prototype.onMouseDown_ = function(e) {
+Blockly.Field.prototype.onMouseDown_ = function(
+  /* eslint-disable no-unused-vars */ e /* eslint-enable no-unused-vars */) {
```
Comment (rachel-fenichel) [inline core/field.js]: If eslint lets you, indent by two more spaces here and everywhere else where you're doing this.

Comment (picklesrus) [reply core/field.js]: Done

File: core/toolbox.js (hunk 1 above)
Comment (picklesrus) [reply core/toolbox.js]: done!

**Comment by rachel-fenichel** [comment]
LG.  If you make this (and following) PRs against the `lint` branch we can keep track of the travis errors better, then merge it all into develop in one go.

**Pull request merged**